source/features/transactions/components/TransactionInfo.tsx

```diff
@@ -31,6 +31,17 @@ const TransactionAddress = (props: { address: string }) =>
     </>
   );
 
+const TransactionAddressMobile = (props: { address: string }) =>
+  props.address.length <= 34 ? (
+    <span>{props.address}</span>
+  ) : (
+    <>
+      <div>{props.address.toString().substring(0, 19)}</div>
```

Comment:
seems like these hardcoded "19" limits the width -> so it won't be full screen :(

Reply:
Yes because there are 2 separated <div> elements with shrinked text. If i remove limit then i can't use word-break on those div elements as it will break the address. This was the only solution to cover it.